contrib/modelci/README.md

```diff
@@ -0,0 +1,47 @@
+# Model CI
+
+A simple Continuous Interation for Models, tracking the overall effect and performance.
```

Comment:
In my mind, the model integration job needs to collect indicators in three aspects. The **model evaluation**(i.e. loss, accuracy), **speed** and **memory cost**. 

For our regression test, we only need to run several batches(say 100 batches) when the training process is stable, then the speed and memory cost data are collected. We also can compare the first several batch losses to validate the model evaluation is correct.

There some problems need to be discussed. 
1. The GPU/CUDA Version difference
Should we consider the different GPU/CUDA machine in regression test? Given a model and a fixed dataset, then some metrics are not changed when you change the training machine and some are changed. For instance, the loss and accuracy regression curves are fixed, but the speed and memory cost will be changed if you use the different version of CUDA and GPU. Our numbers on `Pascal architecture GPU` make no sense on other version GPUs. 

2. The mini batches size difference
For the online learning job or saving training resource, they need small batch size in training models. But for the training speed, they may need big batch size. And the convergence curve is different and the training speed cannot be referenced. Will we consider the different batch size in regression test?

3. The training/inference difference
Currently, most users care more about the inference performance, because online service needs to ensure the performance. The inference is totally different with the training phase, will we consider that in regression test?

Reply:
reference https://github.com/Superjomn/Paddle/blob/254fee8d86bc7872f4b392f5d0dc46c2a5bcc0cf/contrib/modelci/README.md#make-factor-tracking-extensible

the factors to track can be extended, in my understanding, the initial implementation will just include some general factors such as `train cost`, `validate cost`, `duration of each batch`, more factors can be added by more people in the late.